delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala

```diff
@@ -311,21 +316,23 @@ final class Storages private (
           logFailureAndContinue(io)
         }
         .compile
-        .drain
-        .hideErrors >> UIO.unit
+        .drain >> UIO.unit
     }
 
-  private def logFailureAndContinue[A](io: IO[StorageRejection, A]): UIO[Unit] =
-    io.mapError(err => logger.warn(err.reason)).attempt >> UIO.unit
+  private def logFailureAndContinue[A](io: IO[A]): IO[Unit] = {
+    io.onError { case err: StorageRejection =>
+      logger.warn(err.reason)
+    }.attemptNarrow[StorageRejection] >> IO.unit
```

Comment:
`.void` also works

Reply:
Right, I knew there was a method for this. I looked for `.unit`